docs/env_configuration.md

```diff
@@ -1,14 +1,18 @@
+## Shared Configuration
```

Comment:
@damassi I've added a section to explain and link to shared configuration process/docs.

Reply:
👀